packages/evolution-frontend/src/services/errorManagement/errorHandling.ts

```diff
@@ -112,3 +112,70 @@ export const logClientSideMessage = async (
         })
     });
 };
+
+const originalConsoleFunctions: {
+    log: typeof console.log;
+    info: typeof console.info;
+    warn: typeof console.warn;
+    error: typeof console.error;
+} = {
+    log: window.console.log,
+    info: window.console.info,
+    warn: window.console.warn,
+    error: window.console.error
+};
+let consoleOverridden = false;
+
+/**
+ * Override the console functions to send data to the server
+ * @param options
+ * @param {function|undefined} options.getInterviewId An optional function to
+ * retrieve the interview ID to send to the server along with the logs.
+ * @returns
+ */
+export const overrideConsoleLogs = ({ getInterviewId }: { getInterviewId?: () => number | undefined } = {}) => {
+    if (consoleOverridden) {
+        return;
+    }
+    const callClientSideMessage = (data: any[], logLevel: 'log' | 'error' | 'warn' | 'info') => {
+        logClientSideMessage(data.length === 1 ? data[0] : data, {
+            interviewId: getInterviewId !== undefined ? getInterviewId() : undefined,
+            logLevel
+        });
+    };
+    // Do not override console.log in dev mode
```

Comment:
console­.log are very verbose (because of the `devLog` function call) and we want to avoid those, but the other calls, we may want them (as it is not mandatory that only devs use dev builds, we can manually build:dev on an instance also and let it run, waiting for more errors)

Reply:
Should this be configurable in the config instead ?
Wondering if log will be too verbose to log in normal case? Or it's verbose only in dev mode?

Reply:
Please add more comment about why this is skip in Dev mode. Right now it looks like a bit arbitrary, so somebody could be tempted to change it